src/components/Sidebar.jsx

```diff
@@ -163,13 +170,15 @@ function Sidebar(props) {
                 </div>
                 <div className="container d-flex flex-column font-barlow fw-medium p-0 pb-4">
                     <h1 className="text-start text-white font-century-gothic fw-bold fs-2 mb-0 ps-4 pb-3">Conta</h1>
-                    <Link
-                        className="text-white text-decoration-none ps-5 py-2"
-                        to={isDashboard ? '/dash/profile' : '/profile'}
-                        onClick={() => closeSidebar()}
-                    >
-                        Perfil
-                    </Link>
+                    {user.role !== 'GUEST' && user.role !== 'ADMIN' && (
+                        <Link
+                            className="text-white text-decoration-none ps-5 py-2"
+                            to={isDashboard ? '/dash/profile' : '/profile'}
+                            onClick={() => closeSidebar()}
+                        >
+                            Perfil
+                        </Link>
+                    )}
```

Comment:
```suggestion
                    {user.role !== 'GUEST' && user.role !== 'ADMIN' ? (
                        <Link
                            className="px-3 py-1 rounded text-decoration-none text-white bg-steel-blue"
                            to={isDashboard ? '/dash/profile' : '/profile'}
                            onClick={() => closeSidebar()}
                        >
                            {user.username}
                        </Link>
                    ) : (
                        <span className="px-3 py-1 rounded text-white bg-steel-blue">{user.username}</span>
                    )}
```
- Apenas sugestão. Geralmente clicar no seu username em interfaces tende a levar para seu perfil. Coincidiu que este elemento já parece um botão;